rs/ic_os/config/src/types.rs

```diff
@@ -61,6 +61,8 @@ pub struct GuestOSConfig {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct ICOSSettings {
+    /// The node reward type determines node rewards
+    pub node_reward_type: Option<String>,
```

Comment:
node_reward_type is Optional in config, even though it's required for SetupOS installation, because nodes on mainnet currently don't have the field, so this way, the update-config.service can leave the value as optional